process_frames.py

```diff
@@ -303,9 +374,24 @@ def run_image(self, img):
             print("Images are more than 10% different. Proceed with OCR.")
             last_played, highlighted_image, annotations = self.run_ocr(img)
             print(f"finished ocr - {last_played} ")
+
+            if translate:
+                if last_played:
+                    print("Running Translation.")
+                    translation = self.run_translation(self.dialogues[last_played], translate)
+                    print("finished translation")
+                else:
+                    translation = ""
+
             self.previous_image = img
             self.last_annotations = annotations
+
+            if translate:
+                return last_played, self.previous_image, highlighted_image, annotations, translation
```

Comment:
this is super messy, returning different number of parameters from the function, always return the same number